src/providers/completer/command.ts

```diff
@@ -21,38 +21,14 @@ export interface CmdItemEntry {
 }
 
 export interface CmdSignature {
-    name: string,
-    args: string
+    name: string, // name without leading `\`
+    args: string // {} for mandatory args and [] for optional args
```

Comment:
You can add `readonly` to the properties.

You can also add `readonly` to the properties of `interface CmdItemEntry` except `snippet`.